docs/concepts/notifications.md

```diff
@@ -79,7 +85,7 @@ notification = mlrun.model.Notification(
     name="notification-1",
     message="completed",
     severity="info",
-    params={"webhook": "<slack webhook url>"}
+    secret_params={"webhook": "<slack webhook url>"}
```

Comment:
maybe change this example to a webhook notification so you can show off both kinds of params?